pywr/solvers/cython_glpk.pyx

```diff
@@ -1003,41 +1051,81 @@ cdef class CythonGLPKEdgeSolver(GLPKSolver):
         if len(cross_domain_cols) > 0:
             self.idx_row_cross_domain = glp_add_rows(self.prob, len(cross_domain_cols))
 
+        self.edge_fixed_cost_arr[...] = 0.0
+        # Calculate fixed costs
+        for some_node in self.all_nodes:
```

Comment:
This is because `node` is type `Node` which is need for the flow constraints, but `some_node` is `AbstractNode` which is any node including storage, aggregated, etc.